packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -18,17 +18,19 @@ export interface TabProps extends HTMLProps<HTMLAnchorElement> {
   onChangeTab?: (event?: React.MouseEvent<HTMLAnchorElement>) => void;
   /** A number rendered next to the text. Usually used to display the number of items in a tab's view. */
   counter?: number | null;
+  suffix?: FC;
 }
 
 export const Tab = React.forwardRef<HTMLAnchorElement, TabProps>(
-  ({ label, active, icon, onChangeTab, counter, className, href, ...otherProps }, ref) => {
+  ({ label, active, icon, onChangeTab, counter, suffix, className, href, ...otherProps }, ref) => {
     const theme = useTheme2();
     const tabsStyles = getTabStyles(theme);
     const content = () => (
       <>
         {icon && <Icon name={icon} />}
         {label}
         {typeof counter === 'number' && <Counter value={counter} />}
+        {suffix && <span>{suffix({})}</span>}
```

Comment:
Not sure at all about this syntax.

Reply:
I think it'd be possible just to do `{suffix}` here.